core/src/main/scala/reftree/diagram/Animation.scala

```diff
@@ -83,6 +83,32 @@ object Animation {
   /** Create an animation builder from a sequence of starting values */
   def startWithSequence[A: ToRefTree](start: Seq[A]) = Builder(start.toVector)
 
+  private implicit class TakeUntilStream[T](s: Stream[T]) {
```

Comment:
If you end up needing this class, I suggest to add `(val s: ...) extends AnyVal` (http://docs.scala-lang.org/overviews/core/value-classes.html)